osf/models/chronos.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+from osf.models.base import BaseModel
+# from osf.utils.fields import NonNaiveDateTimeField
+from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
+from osf.utils.workflows import ChronosSubmissionStatus
+
+
+class ChronosJournal(BaseModel):
+    name = models.TextField(null=False, blank=False)
+    journal_id = models.TextField(unique=True, null=False, blank=False)
+
+    raw_response = DateTimeAwareJSONField(null=False, blank=False)
+
+    def __repr__(self):
+        return '<{}({})>'.format(
+            self.__class__.__name__,
+            self.name,
+        )
+
+
+class ChronosSubmission(BaseModel):
+    publication_id = models.TextField(null=False, blank=False, unique=True)
+
+    journal = models.ForeignKey(ChronosJournal, null=False, blank=False)
+    preprint = models.ForeignKey('osf.PreprintService', null=False, blank=False)
+
+    # TODO Should be part of the unique?
```

Comment:
Probably not; we don't want a different user trying to submit the same thing to the same journal if another user already has